lib/table.h

```diff
@@ -27,6 +27,17 @@
 DECLARE_MTYPE(ROUTE_TABLE)
 DECLARE_MTYPE(ROUTE_NODE)
 
+/* routing table identifiers
+ *
+ */
+#define RT_TABLE_ID_LOCAL                  255
```

Comment:
These are linux ideas of what some table values are, why do they belong in table.h to expose to theworld?  Additionally why are these part of this commit?  Seems tangential at best

Reply:
agree.
I will put it in table_manager.c, and table_manager.c will have some macros to corner different oses.